miner/worker.go

```diff
@@ -445,45 +438,6 @@ func (w *worker) mainLoop() {
 			}
 			w.commitNewWork(req.interrupt, req.noempty, req.timestamp)
 
-		case ev := <-w.chainSideCh:
```

Comment:
Instead of removing this whole case, can you still include it, but replace the case's block with an error log for now.   I think we should dig deeper on the chainsidech and see if that can be removed altogether.